### DISCUSSION TITLE
How to move a file alone, without its folder tree?

### DISCUSSION THREAD

**Discussion by dimailer** [WinMerge/winmerge#1768] [General]
I have two structures to compare.

Structure 1:
C:\parent_folder\file.txt

Structure 2:
D:\parent_folder\child_folder\file.txt

I want Structure 2 to be identical to Structure 1, so I do:
D:\parent_folder\child_folder\file.txt > RMB > Move > Right to ... > D:\parent_folder\

And I expect to get:
D:\parent_folder\file.txt

Instead, I get:
D:\parent folder\parent folder\child folder\file.txt

WinMerge moved not just file.txt, but all of its folder tree as well.

Is there a way to move just file.txt?

Thanks!

**Comment by sdottaka** [comment]
Unfortunately there is currently no way to move just the file.txt.
Right-click file.txt on the D drive side and click Open Right → Open Parent Folder menu item to open the parent folder in Explorer.
Similarly, you can open the parent folder of file.txt on the C drive side with Explorer, so use them to move the file.

![image](https://user-images.githubusercontent.com/98126/226146626-943470be-0617-4d23-898f-479b537fbc89.png)

**Reply by dimailer** [reply]
Thank you
